modules/transactionpool/update.go

```diff
@@ -185,15 +187,16 @@ func (tp *TransactionPool) ProcessConsensusChange(cc modules.ConsensusChange) {
 		tp.log.Println("NOTE: Upgrading tpool database to support consensus change verification.")
 		resetSanityCheck = true
 	} else if err != nil {
-		tp.log.Println("ERROR: Could not access recentID from tpool.")
+		tp.log.Println("ERROR: Could not access recentID from tpool:", err)
+		build.Critical("ERROR: Could not access recentID from tpool:", err)
```

Comment:
does `tp.log.Critical` do the same thing?

Reply:
I bet it does :P